src/main/java/com/VetApp/PurrgentCare/services/AccountServiceInterface.java

```diff
@@ -1,26 +1,25 @@
 package com.VetApp.PurrgentCare.services;
 
+import com.VetApp.PurrgentCare.dtos.AccountRequest;
 import com.VetApp.PurrgentCare.dtos.AccountResponse;
 import com.VetApp.PurrgentCare.dtos.AssociatePeopleWithAccountRequest;
-import com.VetApp.PurrgentCare.models.Account;
-import com.VetApp.PurrgentCare.models.Person;
 
 import java.util.List;
 
 public interface AccountServiceInterface {
 
 
-    Account getAccount(Integer personId);
+    AccountResponse getAccount(Integer personId);
 
-    List<Account> getAllAccounts();
+    List<AccountResponse> getAllAccounts();
 
-    Account addAccount(Account account);
+    AccountResponse addAccount(AccountRequest accountRequest);
 
     void deleteAccount(Integer personId);
 
-    Account updateAccount(Account account, Integer personId);
+    AccountResponse updateAccount(AccountRequest accountRequest, Integer personId);
```

Comment:
why does this have personId as an input?
Copy-pasta?

Reply:
Must be a few of them have personid, main is the same way.
getAccount, deleteAccount, and updateAccount.

I'll fix the naming.